### PR TITLE
test program: print j nodes with width 3 and leading zeroes

### PR DESCRIPTION
**Pull request by TheDorkKnight**
so that they have consistent width for the printout

### CODE REVIEW

**Comment by TheDorkKnight** [comment]
@rafuzee I've added you as a reviewer on this pull request. This is a simple example of how you can collaborate with other developers via github.

In this pull request, we make it so that nodes print with width `3`, because our maximum node number is on the order of hundreds. That way we get:

```
j001 blah blah blah
j002 blah blah blah
...
j188 blah blah blah
```

and they all line up nicely for good readability.

File: src/main.cpp [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #include <iostream>
+#include <iomanip>
```
Comment (rafuzee) [inline src/main.cpp:2]: Hey Jake,
What is this MAN... Do we need it. I hate verbose code bro...

File: src/main.cpp [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 #include <wl.h>
 
 void print_id(wl::JackNode::ID id) {
-	std::cout << static_cast<int>(jid_to_label(id));
+	std::cout << std::setfill('0') << std::setw(3) << static_cast<int>(jid_to_label(id));
```
Comment (rafuzee) [inline src/main.cpp:6]: beautiful i would never have thought of this.

**Review by rafuzee — COMMENTED** [review]
What wonderful changes...Almost as beautiful as my face.

File: src/main.cpp (hunk 1 above)
Comment (TheDorkKnight) [reply src/main.cpp:2]: That is where `std::setfill` and `std::setw` are defined, so we need it now

**Review by rafuzee — APPROVED** [review]

**Pull request merged**